lib/daru/view.rb

```diff
@@ -123,5 +123,6 @@ def dependent_script(lib=:nyaplot)
         end
       end
     end
-  end # view end
+    # view end
```

Comment:
That Rubocop error advised you against any annotations of `end` ("this is the end of this or that module"), which is bad taste (because you always can see it by indentation), moving "this is the end" comment to the next line is not a proper fix :)

Reply:
Okay! I will remove it . Thanks for suggestion @zverok